app/assess/templates/assessor_tasklist.html

```diff
@@ -17,15 +17,15 @@
     </div>
 
     {% if flag and g.user.highest_role in ("ASSESSOR", "LEAD_ASSESSOR") %}
-        {{ assessment_flag(flag, flag_user_info) }}
+        {{ assessment_flag(flag) }}
```

Comment:
Don't we need the user info here?

Reply:
Yup, removed temporarily to test, thought I put it back!